py/plugins/google-genai/src/genkit/plugins/google_genai/google.py

```diff
@@ -204,8 +212,43 @@ def _resolve_embedder(self, ai: GenkitRegistry, name: str) -> None:
         ai.define_embedder(
             name=googleai_name(_clean_name),
             fn=embedder.generate,
+            metadata=default_embedder_info(_clean_name),
+            config_schema=EmbedContentConfig,
         )
 
+    def list_actions(self) -> list[dict[str, str]]:
```

Comment:
Could you please cache this. This method is quite slow and there's not need to re-fetch actions every time. Dev UI is restarted often, so it's OK making devs restart the dev ui to get new models.